lib/mapbox.rb

```diff
@@ -78,7 +78,7 @@ def self.uri_encode(params)
   end
 
   def self.url_encode(key)
-    URI.escape(key.to_s, Regexp.new("[^#{URI::PATTERN::UNRESERVED}]"))
```

Comment:
I wasn't really sure if removing this was safe. Tried running tests locally and they did pass:

```console
$ bundle exec ruby -Itest test/all_tests.rb

[Coveralls] Set up the SimpleCov formatter.
[Coveralls] Using SimpleCov's default settings.
Loaded suite test/all_tests
Started
........................
Finished in 4.753479 seconds.
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
24 tests, 29 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
5.05 tests/s, 6.10 assertions/s
[Coveralls] Outside the CI environment, not sending data.
```